crates/cibse_guide_e/src/chapter_10/equation_10_10.rs

```diff
@@ -0,0 +1,38 @@
+pub mod openfire_runner;
+
+pub fn limiting_velocity_10_10(g: f64, h: f64, t_f: f64, t_0: f64) -> f64 {
```

Comment:
```suggestion
pub fn limiting_velocity(g: f64, h: f64, t_f: f64, t_0: f64) -> f64 {
```

Once you have separated the equations into their own modules, you no longer need to clarify that it is 10_10 in the function name